dask/array/tests/test_array_core.py

```diff
@@ -617,6 +617,22 @@ def test_store():
     assert raises(ValueError, lambda: store([at, bt], [at, bt]))
 
 
+def test_to_hdf5():
+    try:
+        import h5py
+    except ImportError:
+        return
```

Comment:
Unittest doesn't work on bare function tests.  I dislike using the `py.test` or `nose` libraries in code (I sometimes use one or the other).  I'll throw up a skip function in utils.